com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -869,7 +870,7 @@ public void OnAfterDeserialize()
 #if UNITY_EDITOR
         static void UpgradeAsset(UniversalRenderPipelineAsset asset)
         {
-            if(asset.k_AssetPreviousVersion < 5)
+            if(asset.k_AssetPreviousVersion < k_MaxAssetVersion)
```

Comment:
Looking at this now, it seems that this needed to be still 5, as this is the version it changed from single renderer to renderer list..